src/components/SlateEditor/plugins/uuDisclaimer/SlateDisclaimer.tsx

```diff
@@ -50,16 +50,13 @@ const SlateDisclaimer = ({ attributes, children, element, editor }: Props) => {
   const { data } = element;
   const [modalOpen, setModalOpen] = useState<boolean>(false);
 
-  const embed: UuDisclaimerMetaData | undefined = useMemo(
-    () =>
-      data
-        ? {
-            status: "success",
-            data: { disclaimerLink: { text: "Disclaimer link", href: "https://ndla.no" } },
-            embedData: { ...data, disclaimer: data?.disclaimer },
-            resource: data?.resource,
-          }
-        : undefined,
+  const embed: UuDisclaimerMetaData = useMemo(
+    () => ({
+      status: "success",
+      data: { disclaimerLink: { text: "Disclaimer link", href: "https://ndla.no" } },
```

Comment:
Så lenge du ikkje har en måte å legge inn artikkel-id er det bedre om denne er tom.
```suggestion
      data: {},
```